chat-plugins/mafia.js

```diff
@@ -512,6 +512,10 @@ class MafiaTracker extends Rooms.RoomGame {
 				break;
 			}
 		}
+		if (let p in this.players) {
```

Comment:
Shouldn’t this be something like `for (const player of Object.keys(this.players))`?